front/app/containers/SiteMap/index.tsx

```diff
@@ -154,216 +155,249 @@ const SiteMap = ({ projects, authUser }: Props) => {
   const hasProjectSubsection =
     archivedSection.current || draftSection.current || currentSection.current;
 
-  return (
-    <Container>
-      <SiteMapMeta />
-      {!loaded && (
-        <Loading>
-          <Spinner />
-        </Loading>
-      )}
-      {loaded && (
-        <PageContent>
-          <StyledContentContainer>
-            <QuillEditedContent>
-              <Title>
-                <FormattedMessage {...messages.siteMapTitle} />
-              </Title>
-
-              <TOC>
-                <Header>
-                  <FormattedMessage {...messages.pageContents} />
-                </Header>
-                <Ul>
-                  <li>
-                    <NavItem
-                      onMouseDown={removeFocusAfterMouseClick}
-                      onClick={scrollTo(homeSection)}
-                    >
-                      <FormattedMessage {...messages.homeSection} />
-                    </NavItem>
-                  </li>
-                  <li>
-                    <NavItem
-                      onMouseDown={removeFocusAfterMouseClick}
-                      onClick={scrollTo(userSpaceSection)}
-                    >
-                      <FormattedMessage {...messages.userSpaceSection} />
-                    </NavItem>
-                  </li>
-                  {!isNilOrError(projects) && (
+  if (!isNilOrError(pages)) {
+    const nonCustomStaticPages = pages.filter((page) => {
+      const showPageConditions: Record<TPageCode, boolean> = {
+        proposals: proposalsEnabled,
+        about: true,
+        faq: true,
+        'terms-and-conditions': true,
+        'privacy-policy': true,
+        custom: false,
+      };
+
+      return showPageConditions[page.attributes.code];
```

Comment:
Another simple option could be
```ts
      const showPageConditions: Record<TPageCode, boolean> = {
        proposals: proposalsEnabled,
        custom: false,
      };

      return showPageConditions[page.attributes.code] ?? true;
  ```
  
  But not a big deal.

Reply:
Have you tried it? ;-) The point here is to be very explicit about each page. I don't think it hurts to have that in some places, rather than have bugs.

Reply:
yes indeed, there was a type error